scout/server/blueprints/variants/controllers.py

```diff
@@ -232,61 +232,63 @@ def get_manual_assessments(variant_obj):
 
     for assessment_type in assessment_keywords:
         assessment = {}
-        if variant_obj.get(assessment_type) is not None:
-            if assessment_type == "manual_rank":
-                manual_rank = variant_obj[assessment_type]
-                assessment["title"] = "Manual rank: {}".format(
-                    MANUAL_RANK_OPTIONS[manual_rank]["description"]
-                )
-                assessment["label"] = MANUAL_RANK_OPTIONS[manual_rank]["label"]
-                assessment["display_class"] = MANUAL_RANK_OPTIONS[manual_rank]["label_class"]
+        if variant_obj.get(assessment_type) is None:
```

Comment:
In this function I've just changed this line, so that I could outdent all the following lines